### PR TITLE
Generalize WriteableMonad

### PR DESCRIPTION
**Pull request by sodic**

### CODE REVIEW

File: waspc/cli/src/Wasp/Cli/Command/WaspLS.hs [hunk 1]
```diff
@@ -23,8 +23,8 @@ data Args = Args
   }
 
 data Options = Options
-  { optionsLogFile :: Maybe FilePath,
-    optionsUseStdio :: Bool
+  { _optionsLogFile :: Maybe FilePath,
+    _optionsUseStdio :: Bool
```
Comment (sodic) [inline waspc/cli/src/Wasp/Cli/Command/WaspLS.hs:27]: Fixes the warning @shayneczyzewski mentioned on slack.

Comment (Martinsos) [reply waspc/cli/src/Wasp/Cli/Command/WaspLS.hs:27]: Got it!
How is it that you prefixed both with _ though?

**Review by Martinsos — APPROVED** [review]
LGTM, nice fix! What happened to the RankNTypes fix that we discussed, how is it that that is not in the PR?

**Comment by sodic** [comment]
> LGTM, nice fix! What happened to the RankNTypes fix that we discussed, how is it that that is not in the PR?

It's in now.

**Pull request merged**